### PR TITLE
Add authentik to SSO

### PR DESCRIPTION
**Pull request by larsl-net**
Thank you for taking the time to work on a PR for Awesome-Sysadmin!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software)
- [x] Software your are submitting is not your own, unless you have a healthy ecosystem with a few contributors (which aren't your sock puppet accounts).
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 80 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [ x] Additions are not already listed at [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted)
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-foss/awesome-sysadmin/issues) or [PRs](https://github.com/awesome-foss/awesome-sysadmin/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

--------------

Please take some time to answer the following questions as best you can:

- **Why is it awesome?**

Easy to set up idp, with self service support. In addition to the standard protocols (oAuth and SAML), outposts are available (Proxy, LDAP or RADIUS) for applications that are not natively SSO-capable.

- **Have you used it? For how long?**

Over a year and a half

- **Is this in a personal or professional setup?**

Personal

- **How many devices/users/services/... do you manage with it?**

User < 20
Services 40

- **Biggest pros/cons compared to other solutions?**

Pros:
- Self service
- Outposts (Proxy and LDAP)
- Customization of the flows (login, user registration, ...)
- MFA (OTP, SMS and WebAuth)

Cons:
- With LDAP  Source no write Back feature (User Changes not write to LDAP)

- **Any other comments about your use case, things you've found excellent, limitations you've encountered... ?**

Currently using authentik for my private setup, but currently planning to use it for a corporate setup as well.

### CODE REVIEW

**Review by lost-RD — CHANGES_REQUESTED** [review]
Authentik belongs on this list, but some changes required:

- capitalise Authentik
- capitalise OAuth

**Comment by larsl-net** [comment]
> Authentik belongs on this list, but some changes required:
> 
> * capitalise Authentik
> * capitalise OAuth

Hi @lost-RD I made the Changes.

**Review by lost-RD — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -436,6 +436,7 @@ _See also: [awesome-selfhosted/Database Management](https://github.com/awesome-s
 **[`^        back to top        ^`](#awesome-sysadmin)**
 
 - [Authelia](https://www.authelia.com/) - The Single Sign-On Multi-Factor portal for web apps. ([Source Code](https://github.com/authelia/authelia)) `Apache-2.0` `Go`
+- [Authentik](https://goauthentik.io/) - Flexible identity provider with support for different protocols (OAuth 2.0, SAML, LDAP and Radius). ([Source Code](https://github.com/goauthentik/authentik)) `Python`
```
Comment (nodiscc) [inline README.md:439]: Missing license tag

```suggestion
- [Authentik](https://goauthentik.io/) - Flexible identity provider with support for different protocols. (OAuth 2.0, SAML, LDAP and Radius). ([Source Code](https://github.com/goauthentik/authentik)) `MIT` `Python`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**